.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -0,0 +1,72 @@
+name: Wheels
+on:
+  push:
+    branches:
+      - master
+
+env:
+  CIBW_BUILD: 'cp37-* cp38-* cp39-*'
+
+  # MacOS specific build settings
+  CIBW_BEFORE_ALL_MACOS: |
+    brew uninstall --force oclint
+    brew install gcc libomp
+
+  # Python build settings
+  CIBW_BEFORE_BUILD: |
+    pip install pybind11
+
+  # Testing of built wheels
+  CIBW_TEST_REQUIRES: numpy scipy pytest pytest-cov pytest-mock flaky
+
+  # Note: breaking up tests so that no test cases hang on Windows, to be reverted
+  CIBW_TEST_COMMAND: |
+    pip install numpy==1.19.5 scipy
+    pl-device-test --device=lightning.qubit --skip-ops -x --tb=short --no-flaky-report
```

Comment:
Are we skipping the `pl-device-test --device lightning.qubit --skip-ops --shots=20000` line? Is this what was meant with the breaking up tests comment?

Reply:
Also, do we skip the local tests (`pytest tests/`) to keep things faster for wheel builds?

Reply:
It would seem so, though I am not sure. As far as I can tell this may be some legacy options that no longer apply (ie hanging tests). Do you think the shots=20000 test would be a good inclusion here too?

As for skipping the `tests` dir, I think that is safe here, as they are tested as part of the pip install build elsewhere. Though, if you'd prefer, we can run these on a merge with master, rather than on a PR by PR basis?

Reply:
> Do you think the shots=20000 test would be a good inclusion here too?

Maybe we can check with @antalszava, who I think fixed the hanging tests last time.

But this looks good to me as-is, we also have `pl-device-test --device lightning.qubit --skip-ops --shots=20000` running in the plugin test matrix.

> As for skipping the tests dir, I think that is safe here, as they are tested as part of the pip install build elsewhere.

Makes sense!